src/app/users/users-update/users-update.component.html

```diff
@@ -58,13 +58,26 @@
           </mat-select>
           <mat-error><planet-form-error-messages [control]="editForm.controls.level"></planet-form-error-messages></mat-error>
         </mat-form-field>
-        <label>File Upload:</label>
-        <input type="file" (change)="bindFile($event)" />
       </div>
       <div>
         <button type="submit" mat-raised-button color="primary" i18n>Update</button>
         <button type="button" mat-raised-button color="warn" (click)="goBack()" i18n>Cancel</button>
       </div>
     </form>
+    <div class="profile-upload-view">
```

Comment:
Rather than create new styles for this, we should use the same styles as the image for UsersProfileComponent

Reply:
"profile-upload-view" style keep image and button at the right of form, I change "profile-upload-image" style to "profile-image-large"(the same style as the image for usersProfileComponent).